specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -1564,7 +1569,11 @@
           "description": "Gets or sets the id of the resource."
         }
       },
-      "x-ms-azure-resource": true,
+      "allOf": [
```

Comment:
If adding proxyResource, please remove "name" and "id" (duplicate)

Reply:
Thanks for catching this. I'll remove it.